### ISSUE TITLE
m_blockcaps: discrepancy in behavior and error message

### ISSUE THREAD

**Issue by BlacklightShining** [inspircd/inspircd#1067]
```
if (((caps * 100) / text.length()) >= percent)
{
	user->WriteNumeric(ERR_CANNOTSENDTOCHAN, "%s :Your message cannot contain more than %d%% capital letters if it's longer than %d characters", c->name.c_str(), percent, minlen);
	return MOD_RES_DENY;
}
```

Please either change the comparison to strictly-greater-than, or change the error message to greater-than-or-equal-to.

**Comment by del6597** [comment]
```
#-#-#-#-#-#-#-#-#-#-#-  BLOCKCAPS CONFIGURATION  -#-#-#-#-#-#-#-#-#-#-#
#                                                                     #
# percent        - How many percent of text must be caps before text  #
#                  will be blocked.   
```

Given the language presented in the example modules.conf file I think it's fair to say the error message should be changed.

**Comment by Shawn-Smith** [comment]
The text in `modules.conf` should be changed as well. From `How many percent of text` to `What percentage of the text`

Can you do that in your pull request #1069 as well please?

**Comment by del6597** [comment]
@Shawn-Smith: updated.

**Comment by attilamolnar** [comment]
Fixed by @del6597 in #1069.
